rotkehlchen/tests/unit/decoders/test_main.py

```diff
@@ -75,6 +75,7 @@ def test_decoders_initialization(ethereum_transaction_decoder: 'EthereumTransact
         'Safemultisig',
         'Stakedao',
         'Sushiswap',
+        'Thegraph',
```

Comment:
In the same file at line 92 you forgot to add the counterparty id. This is why the only single test failed